### ISSUE TITLE
Area planted needs to be confirmed

### ISSUE THREAD

**Issue by mstenta** [mstenta/farm_nfa#35]
> 4. “Area planted needs to be confirmed”
> “This is what we said, this is what we did - different?”
> Workflow - someone else will need to confirm that it was done
> Andrew will have a meeting to figure out needs

**Comment by LeScruf** [comment]
Yes - this will need to be trackable as often what is planned is not what is carried out ...

**Comment by pcambra** [comment]
There might be an issue if it's not done in the same year.
